plugin/path_checkouts.go

```diff
@@ -74,3 +408,20 @@ func (b *backend) operationReserveStatus(ctx context.Context, req *logical.Reque
 		Data: respData,
 	}, nil
 }
+
+func canCheckIn(req *logical.Request, checkOut *CheckOut, disableCheckInEnforcement bool) bool {
+	if disableCheckInEnforcement {
+		return true
+	}
+	if checkOut.BorrowerEntityID != "" && req.EntityID != "" {
+		if checkOut.BorrowerEntityID == req.EntityID {
+			return true
+		}
```

Comment:
Can use `return checkOut.BorrowerEntityID == req.EntityID`

Reply:
Well, if they don't pass that check, I'd still like to fall down to the next check and let the caller succeed if they have the same client token as was used to check it out. Because that does verify that they're the same person as who checked it out, just in a different way.

Reply:
You're right. This is fine as is.